core/tests/epoch_accounts_hash.rs

```diff
@@ -656,6 +657,7 @@ fn test_epoch_accounts_hash_and_warping() {
 // https://github.com/rust-lang/rust/pull/88582
 // https://github.com/jhpratt/rust/blob/727a4fc7e3f836938dfeb4a2ab237cfca612222d/library/core/src/num/uint_macros.rs#L1811-L1837
 const fn next_multiple_of(lhs: u64, rhs: u64) -> u64 {
+    #![allow(clippy::integer_arithmetic)]
     match lhs % rhs {
         0 => lhs,
```

Comment:
Agreed that this is fine to leave given that this is a workaround until `next_multiple_of` stabilizes